src/test/java/io/strimzi/kafka/metrics/KafkaMetricsCollectorTest.java

```diff
@@ -50,33 +50,25 @@ public void testCollect() {
         metrics = collector.collect();
         assertTrue(metrics.isEmpty());
 
-        // Adding a non-numeric metric converted
-        KafkaMetric nonNumericMetric = buildNonNumericMetric("name", "group");
-        collector.addMetric(nonNumericMetric);
-        metrics = collector.collect();
-        assertEquals(1, metrics.size());
-        assertEquals("kafka_server_group_name", metrics.get(0).name);
-        assertEquals(1, metrics.get(0).samples.size());
-        assertEquals(1.0, metrics.get(0).samples.get(0).value, 1.0);
-        assertTrue(metrics.get(0).samples.get(0).labelNames.contains("value"));
-
         // Adding a metric that matches the allowlist
         collector.addMetric(buildMetric("name", "group", 1.0));
         metrics = collector.collect();
+        Collector.MetricFamilySamples getMetrics = metrics.get(0);
```

Comment:
Can we find a better variable name? Maybe `metricFamilySamples`

Reply:
Changed